scripts/linters/js_ts_linter.py

```diff
@@ -845,6 +845,82 @@ def _check_constants_declaration(self):
 
         return summary_messages
 
+    def _check_comments(self):
```

Comment:
@Hudda Any specific reason this was not added via eslint new rule/extension like the way we do for pylint? I think the eslint approach would be faster, right? (not sure!)

Reply:
Done. We only have a rule for Capital letter in eslint and not for a period at the end of comment. So we still need a custom check for that.

Reply:
Why don't we write our own custom rules (like the way we write for pylint?) I think that would really be fast, any specific reason for not writing a custom eslint rule for this?

Reply:
Right now, it is just one rule, we will add this if we need more rules, Probably in the 3rd milestone.